src/libs/routes/errorHandler.ts

```diff
@@ -0,0 +1,11 @@
+const errorHandler = ((err , req , res , next) => {
+    console.log('Error', err);
+    res.send(
+        {
+            error : err.error,
```

Comment:
there should be an optional error if you get any
 error: error || 'undefined'

Reply:
done